benchmarks/benchmarks/stats.py

```diff
@@ -249,6 +249,84 @@ def time_distribution(self, dist_name, method):
         self.method(*self.args, **self.kwds)
 
 
+class TrackContinuousRoundtrip(Benchmark):
+    # Benchmarks that track a value(s) for every distribution can go here
+    param_names = ['dist_name']
+    params = sorted(list(set([d[0] for d in distcont])))
+    dist_data = dict(distcont)
+
+    def setup(self, dist_name):
+        # MOST SETUP CODE COPIED FROM ABOVE `DistributionsAll` BENCHMARK #
```

Comment:
Setup code from `DistributionsAll` is very long but this is very short, so it could be more clear to explain what is being sourced from there. Up to you!
```suggestion
        #  Distribution setup follows `DistributionsAll` benchmark.
```

Reply:
Resolved.